blueoil/datasets/delta_mark.py

```diff
@@ -82,32 +81,11 @@ def __init__(
             **kwargs,
         )
 
-        self.is_shuffle = is_shuffle
-
         self.path = {
             "json": os.path.join(self.data_dir, json_file),
             "dir": os.path.join(self.data_dir, image_dir)
         }
 
-    @property
-    def indices(self):
-        if not hasattr(self, "_indices"):
-            if self.subset == "train" and self.is_shuffle:
-                self._indices = shuffle(range(self.num_per_epoch), seed=self.seed)
-            else:
-                self._indices = list(range(self.num_per_epoch))
-
-        return self._indices
-
-    def _get_index(self, counter):
-        return self.indices[counter]
-
-    def _shuffle(self):
```

Comment:
This method is called in `dataset_iterator.py` but I think it's not needed. Could you refactor it also?